lib/src/screens/feed/create_screen.dart

```diff
@@ -194,6 +200,61 @@ class _CreateScreenState extends State<CreateScreen> {
       ),
     );
 
+    Widget pollOptionsWidget() => Column(
+      children: [
+        ..._pollOptions.map(
+          (option) => Padding(
+            padding: const EdgeInsets.all(5),
+            child: Row(
+              children: [
+                Expanded(child: TextEditor(option)),
+                IconButton(
+                  onPressed: () => setState(() {
+                    _pollOptions.remove(option);
+                  }),
+                  icon: const Icon(Symbols.delete_rounded),
+                ),
+              ],
+            ),
+          ),
+        ),
+        ElevatedButton(
+          onPressed: () => setState(() {
+            _pollOptions.add(
+              TextEditingController(text: 'Option ${_pollOptions.length}'),
+            );
+          }),
+          child: Text(l(context).addChoice),
+        ),
+        ListTile(
+          title: Text(l(context).pollMode),
+          onTap: () async {
+            final mode = await pollMode(
+              context,
+            ).askSelection(context, _pollMode);
+            if (mode == null) return;
+            setState(() {
+              _pollMode = mode;
+            });
+          },
+          trailing: Text(pollMode(context).getOption(_pollMode).title),
+        ),
```

Comment:
It seems like we could replace this section with a simple toggle switch, something like: "Allow multiple selecitons".